packages/grafana-ui/src/components/CallToActionCard/CallToActionCard.tsx

```diff
@@ -34,7 +34,7 @@ const getStyles = (theme: GrafanaTheme2) => ({
     label: call-to-action-card;
     padding: ${theme.spacing(3)};
     background: ${theme.colors.background.secondary};
-    border-radius: ${theme.shape.borderRadius(2)};
+    border-radius: ${theme.shape.borderRadius(1)};
```

Comment:
I guess we don't need the 1 here?
(Looks to me like it defaults to 1 in case it doesn't have a parameter [here](https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/themes/createShape.ts#L15)).

```suggestion
    border-radius: ${theme.shape.borderRadius()};
```